kuka_rsi_driver/src/robot_manager_node_rsi_only.cpp

```diff
@@ -135,10 +144,24 @@ RobotManagerNodeRsi::on_deactivate(const rclcpp_lifecycle::State &)
 
   // Stop RT controllers
   // With best effort strictness, deactivation succeeds if specific controller is not active
+  std::vector<std::string> deactivate_controllers = {
+    kuka_drivers_core::JOINT_STATE_BROADCASTER, this->position_controller_name_};
+  if (use_gpio_)
+  {
+    deactivate_controllers.push_back(kuka_drivers_core::GPIO_CONTROLLER);
+  }
+
+  const bool controller_activation_successful = kuka_drivers_core::changeControllerState(
+    change_controller_state_client_, deactivate_controllers, {});
+  if (!controller_activation_successful)
```

Comment:
is there a reason for adding this extra bool? I would prefer if you did not create the variable, but directly  use the changeControllerState output
(I see that in the rsi-eki file, the original code also contained the bool, please also change that + for deactivation as well)